metrics/lib/common.bash

```diff
@@ -42,6 +42,27 @@ function save_results(){
 	bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
 }
 
+# This function checks existence of commands.
+# They can be received standalone standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
+# check_cmds "${cmd[@]}"
+function check_cmds()
+{
+	set +e
```

Comment:
well, about "function" keyword, I was following a convention in that file,also I would like to keep "function" keyword in order to avoid "alias" collisions